### DISCUSSION TITLE
Есть ли методы увеличение площади покрытия Bluetooth?

### DISCUSSION THREAD

**Discussion by Ya-Al-Ya** [TionAPI/HA-tion#125] [Q&A]
Добрый день.
Интеграция великолепно работает, спасибо!
Но есть один бризер (Tion Lite) до которого «не добивает» покрытие модуля Bluetooth, подключенного через удлинитель к USB порту Raspberry pi (Home Assistant 2023.10.3). Может есть идеи или методики как можно увеличить зону покрытия Bluetooth для этой интеграции? Как идея - через локальный Tuya-шлюз или Bluetooth Proxy (ESPHome).

**Comment by IATkachenko** [comment]
Кажется ESP32 можно использовать как BTLE прокси.
Но если есть ESP32, то я бы рекомендовал использовать https://github.com/dentra/esphome-tion
